Gemfile

```diff
@@ -5,6 +5,8 @@ ruby "3.2.2"
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
 gem "rails", "~> 7.1.3", ">= 7.1.3.4"
 
+gem "byebug"
```

Comment:
Only use double strings when we are doing string interpolation. Use single quotes everywhere else.